src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -350,13 +349,14 @@ private static IEnumerable<CompletionItem> GetUserDefinedTypeCompletions(Semanti
             || syntax is IntegerLiteralSyntax
             || (syntax is StringSyntax @string && @string.TryGetLiteralValue() is string literal)
             || syntax is UnionTypeSyntax
-            || (syntax is ObjectTypeSyntax objectType && objectType.Properties.All(p => IsTypeLiteralSyntax(p.Value)));
+            || (syntax is ObjectTypeSyntax objectType && objectType.Properties.All(p => IsTypeLiteralSyntax(p.Value)))
+            || (syntax is TupleTypeSyntax tupleType && tupleType.Items.All(i => IsTypeLiteralSyntax(i.Value)));
 
```

Comment:
Are there completion tests covering this logic / these changes?

Reply:
Added in c4efb017fc9c1adf9bf073b02e12f896a54df296